python/dgllife/utils/featurizers.py

```diff
@@ -2081,3 +2155,171 @@ def __init__(self, bond_data_field='e', self_loop=False):
                                                              Chem.rdchem.BondStereo.STEREOZ,
                                                              Chem.rdchem.BondStereo.STEREOE])]
             )}, self_loop=self_loop)
+
+class PAGTN_EdgeFeaturizer(object):
+    """The bond featurizer used in PAGTN
+
+    PAGTN is introduced in
+    `Path-Augmented Graph Transformer Network. <https://arxiv.org/abs/1905.12712>`
+
+    We build a complete graph and The Edge features include:
+    * **Shortest path between two nodes in terms of bonds**. Each bond is a One hot
+        encoding of bond type, conjugacy, ring membership
```

Comment:
I don't understand why do we need "Each bond is a One hot encoding of bond type, conjugacy, ring membership"

Reply:
I have used the term one hot encoding inappropriately and I will change it. This is what I had meant
Each path between two nodes is a sequence of Rdkit Bonds datatype. The feature for each bond is a concatenation of `bond_type_one_hot`, ` bond_is_conjugated`, `bond_is_in_ring` . And all these path features are concatenated as a whole in the end.

Reply:
How about "Shortest path between two nodes. To encode the path, we encode each bond on the path and concatenate their encodings. The encoding of a bond contains information about the bond type, whether the bond is conjugated and whether the bond is in a ring."

Reply:
Yeah this sounds better